src/popup/components/AccountItem.vue

```diff
@@ -74,10 +82,16 @@ export default defineComponent({
   }
 
   .address {
-    @extend %face-mono-10-medium;
-
     text-align: left;
     word-break: break-all;
+
+    &.rg {
```

Comment:
```suggestion
    &.md {
```
I think `md` will make more sense